protocols/indexer/contracts/Indexer.sol

```diff
@@ -179,10 +179,10 @@ contract Indexer is IIndexer, Ownable {
       "INTENT_DOES_NOT_EXIST");
 
     // Unset the intent on the market.
-    markets[_makerToken][_takerToken].unsetIntent(msg.sender);
+    require(markets[_makerToken][_takerToken].unsetIntent(msg.sender));
```

Comment:
given that market sends back false if there are no intents. I don't think reverting is the correct behavior in this instance.